.github/dependabot.yml

```
@@ -0,0 +1,24 @@
+version: 2
+updates:
+- package-ecosystem: "gomod"
+  directory: "/"
+  schedule:
+    interval: "weekly"
+
+- package-ecosystem: "docker"
+  directory: "/"
+  schedule:
+    interval: "weekly"
```

Usually we only bump direct dependencies.
```suggestion
  schedule:
    interval: "weekly"
  allow:
    - dependency-type: direct    
```

---

Added to the `gomod` section.